tests/test_utils.py

```diff
@@ -288,3 +290,9 @@ def test_autocomplete_order_parameters():
     assert order_parameters["params"]["existing_param1"] is not None
     assert order_parameters["params"]["geometry"] is None
     assert order_parameters["params"]["acquisitionMode"] is None
+
+
+@patch("importlib.metadata.version", return_value="0.33.0")
```

Comment:
NIT: this can be confusing and feel like it needs to be updated. I'd still recommend to set it to 0.0.0 or "some-version"